core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -456,6 +456,8 @@
       "The amount of time an assignment can run before the server will print a"
           + " warning along with the current stack trace. Meant to help debug stuck"
           + " assignments"),
+  TSERV_REPLICATION_ENABLED("tserver.replication.services.enabled", "false", PropertyType.BOOLEAN,
```

Comment:
Look at the services that the Master starts up for replication, it is a bit trickier.  And I am not sure how much we save since there should only be one Master running versus X TabletServers.  I think making it a tserver property makes the most sense here.  There may also be a use case where you want certain tservers to participate in replication and disabled on others.

Reply:
Even if you are not going to touch replication in other processes, I would keep the name `replication.enabled`.  It's either enabled or it's not. If can confusing if we have other similar properties like `master.replication.enabled` as you don't want users enabling it in tserver but not the master.

Reply:
>  It's either enabled or it's not

This.

Either replication is on (and all the necessary services are running) or it's off (and none of the services are running). I thought you were advocating to moving 100% of the way to the latter with this change, @milleruntime.

Reply:
Ideally, yes.

Reply:
As I understand it, the current design doesn't make a single `replication.enabled` option as simple to do as it first appears. The elements in the master that are replication related appear to be part of the replica "send" code path.... the elements in the tserver that we're talking about appear related to the "AccumuloReplicaSystem", which is specifically a tserver feature for handling received replication data at the destination (the replica "receive" code path). It should be entirely possible to enable these features independently (to support basically any use case that isn't bi-directional replication between two Accumulo instances).

Reply:
> the current design doesn't make a single replication.enabled option as simple to do as it first appears.

I'm not sure what you think is untenable here. There are multiple services that are started, in different processes, but it all uses the single configuration option: a global on/off.

> It should be entirely possible to enable these features independently (to support basically any use case that isn't bi-directional replication between two Accumulo instances).

The current implementation is definitely not intended to be used in this fashion. I'd caution against trying to separate these as it's going to open a can of worms. If nothing else, if this is an itch you want to scratch, do that on its own to avoid conflating these issues.